setup.cfg

```diff
@@ -0,0 +1,34 @@
+[metadata]
+name = eedl
+description= Code to export arbitrarily large areas of data from Earth Engine to local storage and optionally process data locally afterward
+author = nickrsan, AdamJCrawford
+license = MIT
+platforms = unix, linux, osx, cygwin, win32
+url = https://github.com/water3d/eedl
+version = 0.1.0
+classifiers =
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+
+[options]
+packages =
+    eedl
+install_requires =
+    gdal
+    rasterstats
+    earthengine-api
+    fiona
+    pandas
+    seaborn>=0.12.0
+    google-cloud-storage
+    setuptools
+    requests
+    pytest
+python_requires = >= 3.8
+package_dir =
+    =src
+zip_safe = no
```

Comment:
We should remove this line. It's no longer needed.

Reply:
I may be missing something, but which line are you referring to?

Reply:
Ahh, sorry - yes, GitHub makes this ambiguous when it's near the end of the file since it shows the full context.

We should remove zip_safe line. It's outdated. I have another comment about the package_dir separately